src/bin/info.rs

```diff
@@ -55,6 +62,30 @@ async fn user_states_example(info_client: &InfoClient) {
     );
 }
 
+async fn user_token_balances_example(info_client: &InfoClient) {
+    let user: String = "0xc64cc00b46101bd40aa1c3121195e85c0b0918d8"
```

Comment:
nit: define address string as a constant and reuse

Reply:
Done